### ISSUE TITLE
LESS mapping doesn't work in Chrome DevTools

### ISSUE THREAD

**Issue by borekb** [pingyhq/pingy-cli#30]
Steps to reproduce (I'm on Windows 10, tried Chrome 58 and 60):

Create an empty folder, `pingy init` in it, choose HTML + LESS, let it scaffold everything.

```
$ pingy init                                                                                                      
0.6.6 (global)                                                                                                    
  ____  _                                                                                                         
 |  _ \(_)_ __   __ _ _   _                                                                                       
 | |_) | | '_ \ / _` | | | |                                                                                      
 |  __/| | | | | (_| | |_| |                                                                                      
 |_|   |_|_| |_|\__, |\__, |                                                                                      
                |___/ |___/                                                                                       
                                                                                                                  
                                                                                                                  
? What document format do you wish to use HTML                                                                    
? What styles format do you wish to use Less                                                                      
? What scripts format do you wish to use JS                                                                       
? Choose the folder name to export compiled files to dist                                                         
This utility will walk you through creating a package.json file.                                                  
It only covers the most common items, and tries to guess sensible defaults.                                       
                                                                                                                  
See `npm help json` for definitive documentation on these fields                                                  
and exactly what they do.                                                                                         
                                                                                                                  
Use `npm install <pkg> --save` afterwards to install a package and                                                
save it as a dependency in the package.json file.                                                                 
                                                                                                                  
Press ^C at any time to quit.                                                                                     
name: (lessmapping)                                                                                               
version: (1.0.0)                                                                                                  
description:                                                                                                      
entry point: (index.js)                                                                                           
test command:                                                                                                     
git repository:                                                                                                   
keywords:                                                                                                         
author:                                                                                                           
license: (ISC)                                                                                                    
About to write to c:\Dev\temp\lessmapping\package.json:                                                           
                                                                                                                  
{                                                                                                                 
  "name": "lessmapping",                                                                                          
  "version": "1.0.0",                                                                                             
  "description": "",                                                                                              
  "main": "index.js",                                                                                             
  "scripts": {                                                                                                    
    "test": "echo \"Error: no test specified\" && exit 1"                                                         
  },                                                                                                              
  "author": "",                                                                                                   
  "license": "ISC"                                                                                                
}                                                                                                                 
                                                                                                                  
                                                                                                                  
Is this ok? (yes)                                                                                                 
√ Pingy scripts added to package.json                                                                             
√ Created .pingy.json                                                                                             
? Do you want Pingy to scaffold the following files for you? Yes                                                  
? The most important question: Tabs or spaces 2 spaces                                                            
√ Site files scaffolded                                                                                           
                                                                                                                  
Ready to install dependencies.                                                                                    
                                                                                                                  
Command that will now be run:                                                                                     
  > npm install --save-dev @pingy/cli less                                                                        
                                                                                                                  
? Run this command now? Yes                                                                                       
lessmapping@1.0.0 c:\Dev\temp\lessmapping                                                                         
+-- @pingy/cli@0.6.6                                                                 
>> LONG NPM OUTPUT HERE <<                             
                                                                                                                  
npm WARN optional SKIPPING OPTIONAL DEPENDENCY: fsevents@^1.0.0 (node_modules\chokidar\node_modules\fsevents):    
npm WARN notsup SKIPPING OPTIONAL DEPENDENCY: Unsupported platform for fsevents@1.1.1: wanted {"os":"darwin","arch
":"any"} (current: {"os":"win32","arch":"x64"})                                                                   
npm WARN lessmapping@1.0.0 No description                                                                         
npm WARN lessmapping@1.0.0 No repository field.                                                                   
√ Dependencies installed                                                                                          
```

Run `pingy dev` and open DevTools. LESS mapping works, so far so good!

![image](https://cloud.githubusercontent.com/assets/101152/26175420/ea72ecb2-3b52-11e7-8eb0-48fc6cd7623b.png)

Clicking on `styles\main.less`, being prompted to add files into workspace:

![image](https://cloud.githubusercontent.com/assets/101152/26175456/080e270a-3b53-11e7-8a38-c47a7034e598.png)

I've done that:

![image](https://cloud.githubusercontent.com/assets/101152/26175546/4f43ca1c-3b53-11e7-97c3-8dc374d77b53.png)

But when I go to Elements again, click `main.less` and edit the color, it's clearly not mapped:

![image](https://cloud.githubusercontent.com/assets/101152/26175610/7ad83e92-3b53-11e7-9394-c03c8f427b74.png)

Trying to create a mapping:

![image](https://cloud.githubusercontent.com/assets/101152/26175666/ae25aa32-3b53-11e7-8188-561d34c085eb.png)

![image](https://cloud.githubusercontent.com/assets/101152/26175692/c7a40d78-3b53-11e7-955a-c147cd52d4d6.png)

But it still doesn't work (is probably not mapped; I don't know how to tell for sure):

![image](https://cloud.githubusercontent.com/assets/101152/26175727/ebfa8d14-3b53-11e7-9f61-5ca752b53ffb.png)

I've also tried the mapping the other way around but it doesn't work either:

![image](https://cloud.githubusercontent.com/assets/101152/26175781/1ace185e-3b54-11e7-8f12-2251da373891.png)

Just to make sure, editing the value in the `.less` file on the disk indeed works:

![image](https://cloud.githubusercontent.com/assets/101152/26175828/485dc864-3b54-11e7-8893-dc0ca972f7ec.png)

Could the backslashes be the problem? For example, this looks slightly suspicious:

![image](https://cloud.githubusercontent.com/assets/101152/26175873/79cb92c8-3b54-11e7-8405-042358d7b66f.png)

**Comment by borekb** [comment]
Switching to `upath` module fixed it for me: https://github.com/pingyhq/pingy-cli/pull/31

**Comment by davej** [comment]
Awesome, thanks! Windows support is admittedly a bit flaky at the moment as I don't use the OS myself.

I'm on holidays right now but I'll merge this and publish a new release in the next few weeks when I'm back home. Thanks!

**Comment by borekb** [comment]
It actually works really well on Windows, I am impressed :) Good job, enjoy your holidays!

**Comment by davej** [comment]
Fixed in v0.6.8: https://github.com/pingyhq/pingy-cli/releases/tag/%40pingy%2Fcli%400.6.8
